content/en/docs/faq/installation/uninstall-pluggable-components.md

```diff
@@ -0,0 +1,219 @@
+
+   {{</ notice >}}
+
+3. To uninstall Logging system including Elasticsearch:
+
+   ```bash
+   kubectl delete crd fluentbitconfigs.logging.kubesphere.io
+   kubectl delete crd fluentbits.logging.kubesphere.io
+   kubectl delete crd inputs.logging.kubesphere.io
+   kubectl delete crd outputs.logging.kubesphere.io
+   kubectl delete crd parsers.logging.kubesphere.io
+   kubectl delete deployments.apps -n kubesphere-logging-system fluentbit-operator
+   helm uninstall elasticsearch-logging --namespace kubesphere-logging-system
+   ```
+
+   {{< notice note >}}
+
+   This operation may cause anomalies in Auditing, Events, and Service Mesh.
+
+   {{</ notice >}}
+
+## Uninstall KubeSphere Events
+
+1. Change the value of `events.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. Run the following command:
+
+   ```bash
+   helm delete ks-events -n kubesphere-logging-system
+   ```
+
+## Uninstall KubeSphere Alerting
+
+1. Change the value of `alerting.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. Run the following command:
+
+   ```bash
+   kubectl -n kubesphere-monitoring-system delete thanosruler kubesphere
+   ```
+
+   {{< notice note >}}
+
+   Notification is installed in KubeSphere v3.1.x by default, so you do not need to uninstall it.
+
+   {{</ notice >}} 
+
+
+## Uninstall KubeSphere Auditing
+
+1. Change the value of `auditing.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. Run the following commands:
+
+   ```bash
+   helm uninstall kube-auditing -n kubesphere-logging-system
+   kubectl delete crd awh
+   kubectl delete crd ar
+   ```
+
+## Uninstall KubeSphere Service Mesh
+
+1. Change the value of `servicemesh.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. Run the following commands:
+
+   ```bash
+   curl -L https://istio.io/downloadIstio | sh -
+   istioctl x uninstall --purge
+   
+   kubectl -n istio-system delete kiali kiali
+   helm -n istio-system delete kiali-operator
+   
+   kubectl -n istio-system delete jaeger jaeger
+   helm -n istio-system delete jaeger-operator
+   ```
+
+## Uninstall Network Policies
+
+For the component NetworkPolicy, disabling it does not require uninstalling the component as its controller is now inside `ks-controller-manager`. If you want to remove it from the KubeSphere console, change the value of `network.networkpolicy.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+## Uninstall Metrics Server
+
+1. Change the value of `metrics_server.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
+
+2. Run the following command:
+
+   ```bash
+   helm delete metrics-server -n kube-system
+   ```
```

Comment:
Perfom the following steps to uninstall Metrics Server for KubeSphere v3.1.x: 
1. Change the value of `metrics_server.enabled` from `true` to `false` in `ks-installer` of the CRD `ClusterConfiguration`.
2. Run the following commands: 
    ```shell
    kubectl delete apiservice v1beta1.metrics.k8s.io
    kubectl -n kube-system delete service metrics-server
    kubectl -n kube-system delete deployment metrics-server
    ```

Reply:
Thanks! Updates made in [this commit](https://github.com/kubesphere/website/pull/1863/commits/a073b38fc68dcd83cc9ca330c57de8933922aee7).